Gemfile

```diff
@@ -11,8 +11,9 @@ gem 'yard', '~> 0.8.7', :require => false
 
 ### deps for rdoc.info
 group :documentation do
-  gem 'redcarpet',     '2.3.0'
-  gem 'github-markup', '1.0.0'
+  gem 'redcarpet', '~> 3.4'
```

Comment:
please use exact pins. We want to pin to some low version, which is the lowest version that we support, such that as new versions get released, we know RSpec still works with old versions of those gems.

I'd say `gem 'redcarpet', '3.4.x'` where x is the highest currently released version in the series, and the same for all other gems in this file.